crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/util/RoundingUtilsTest.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.laa.crime.meansassessment.util;
+
+import org.junit.Test;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Modifier;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class RoundingUtilsTest {
+    @Test
+    public void testRoundingUtilConstructorIsPrivate() throws NoSuchMethodException {
+        assertEquals("There must be only one constructor", 1,
+                RoundingUtils.class.getDeclaredConstructors().length);
+        Constructor<RoundingUtils> constructor = RoundingUtils.class.getDeclaredConstructor();
+        assertTrue(Modifier.isPrivate(constructor.getModifiers()));
```

Comment:
Is it really worth testing whether a constructor is private? This seems superfluous to me.

Reply:
This was added to improve the test coverage.